gum/arch-arm/gumthumbwriter.c

```diff
@@ -788,6 +788,55 @@ gum_thumb_writer_put_ldr_reg_reg_offset (GumThumbWriter * self,
       GUM_THUMB_MEMORY_LOAD, dst_reg, src_reg, src_offset);
 }
 
+gboolean
+gum_thumb_writer_put_vldr_reg_reg_offset (GumThumbWriter * self,
+                                         arm_reg dst_reg,
+                                         arm_reg src_reg,
+                                         gsize src_offset)
+{
+  GumArmRegInfo src_reg_info;
+  GumArmRegInfo dst_reg_info;
+
+  guint32 code = 0xed900a00;
```

Comment:
```suggestion
  guint32 insn = 0xed900a00;
```

(For consistency with how we've named it in other functions.)